build_ext/build_ext/lint.py

```diff
@@ -88,8 +90,11 @@ class RpmLint(BaseCommand):
     description = "run rpmlint on spec files"
 
     def run(self):
+        files = subprocess.run(["git", "ls-files", "--full-name"], capture_output=True).stdout
+        files = files.decode().splitlines()
+        files = [x for x in files if x.endswith(".spec")]
```

Comment:
this is basically 1400a3e2ec537e76801027bc6c9b82573c82bfef; in case we want it, simply backport as own commit, so it is clear it is that commit

otherwise, the actual change needed of 4af3f18f50c2e30090f7ae28df2e8e45c695348c is removing `--file=rpmlint.config` from the parameters of the `rpmlint` process, and that should be enough